Main.cs

```diff
@@ -279,80 +261,85 @@ public string GetTranslatedPluginDescription()
 
         public List<Result> LoadContextMenus(Result selectedResult)
         {
-            SearchResult record = selectedResult.ContextData as SearchResult;
-            List<Result> contextMenus = new List<Result>();
-            if (record == null) return contextMenus;
+            if (selectedResult.ContextData is not SearchResult record)
+                return new List<Result>();
 
-            List<ContextMenu> availableContextMenus = new List<ContextMenu>();
-            availableContextMenus.AddRange(GetDefaultContextMenu());
-            availableContextMenus.AddRange(_settings.ContextMenus);
+            var icoPath = record.Type == ResultType.File ? "Images\\file.png" : "Images\\folder.png";
 
-            if (record.Type == ResultType.File)
+            List<Result> contextMenus = new List<Result>
             {
-                foreach (ContextMenu contextMenu in availableContextMenus)
+                new()
                 {
-                    var menu = contextMenu;
-                    contextMenus.Add(new Result
+                    Title = _context.API.GetTranslation("flowlauncher_plugin_everything_open_containing_folder"),
+                    Action = _ =>
                     {
-                        Title = contextMenu.Name,
-                        Action = _ =>
-                        {
-                            var parentPath = Directory.GetParent(record.FullPath);
-
-                            if ((menu.Argument.Trim() == Settings.DirectoryPathPlaceHolder || string.IsNullOrWhiteSpace(menu.Argument)) && _settings.ExplorerPath.Trim() == Settings.Explorer)
-                                menu.Argument = Settings.DefaultExplorerArgsWithFilePath;
-
-                            string argument = menu.Argument.Replace(Settings.FilePathPlaceHolder, '"' + record.FullPath + '"')
-                                .Replace(Settings.DirectoryPathPlaceHolder, '"' + parentPath.ToString() + '"');
-
-
-                            try
-                            {
-                                Process.Start(menu.Command, argument);
-                            }
-                            catch
-                            {
-                                _context.API.ShowMsg(string.Format(_context.API.GetTranslation("flowlauncher_plugin_everything_canot_start"), record.FullPath), string.Empty, string.Empty);
-                                return false;
-                            }
-                            return true;
-                        },
-                        IcoPath = contextMenu.ImagePath
-                    });
+                        _context.API.OpenDirectory(Path.GetDirectoryName(record.FullPath), record.FullPath);
```

Comment:
i dont remember OpenDirectory API has try catch, if not, please put one here.

Reply:
probably we should add the check there?

Reply:
yep, true. if you not going to add here please make sure you do over there :)

Reply:
probably just let it throw so user notice they has done something wrong?